src/components/calcite-pick-list-item/calcite-pick-list-item.scss

```diff
@@ -1,11 +1,16 @@
 :host {
-  display: flex;
   align-items: center;
-  padding: 12px;
-  background-color: white;
-  box-shadow: 0px 2px 3px 0px rgba(0, 0, 0, 0.75);
-  margin-bottom: 12px;
+  background-color: var(--calcite-app-background);
+  color: var(--calcite-app-foreground);
   cursor: pointer;
+  display: flex;
+  margin: 0 0 var(--calcite-app-cap-spacing-minimum) 0;
```

Comment:
would margin-bottom work or do we need to zero out the others?

Reply:
👍